### ISSUE TITLE
tie fighter reversed

### ISSUE THREAD

**Issue by bleachorange** [Valadian/ArmadaTTS#12]
The tie fighters are reversed with their orientation and buttons compared to other squadrons. they are always flying backwards. just need to 180 them or their buttons, I guess.

**Comment by Valadian** [comment]
Ha. I remember that one. Consider it a feature!

I will flip around their model.

**Comment by bleachorange** [comment]
yeah, its not a big deal, but every time i run across it it just bugs me.
